web/cobrands/merton/base.scss

```diff
@@ -277,3 +277,9 @@ ol.big-numbers {
 }
 /* end of fixes for around page */
 
+table.googlecookies {
+    th, td {
+        width: 1%;
```

Comment:
I think I understand what this does but not why/how - could you add a comment explaining it please?